front-end/src/store/store.jsx

```diff
@@ -1,7 +0,0 @@
-import { configureStore } from '@reduxjs/toolkit';
```

Comment:
Was there a reason this got deleted?

Reply:
nvm, I see that you re-made the file!

Reply:
I'm so confused now @.@ I still see that line on my vs code...hmm, let me try pushing again.